src/coreclr/vm/CMakeLists.txt

```diff
@@ -660,6 +660,19 @@ if(CLR_CMAKE_TARGET_WIN32)
 
 endif(CLR_CMAKE_TARGET_WIN32)
 
+if(CLR_CMAKE_HOST_UNIX)
```

Comment:
Instead of doing this I would remove the `rcwrefcache.cpp` from its current position in this file and add it to the the existing location for `FEATURE_COMWRAPPERS` - https://github.com/dotnet/runtime/blob/33452e9c11330528d198d82b0afd8a2ce6502e47/src/coreclr/vm/CMakeLists.txt#L555-L577

Reply:
Moved rcwrefcache.cpp in FEATURE_COMWRAPPERS location; rcwrefcache.cpp is even appended in another location for WIN32, could it be a problem in case of  FEATURE_COMWRAPPERS enabled for windows?

Reply:
Yes. The `rcwrefcache.*` files and types should all be behind `FEATURE_COMWRAPPERS`. Those types are only used with the `ComWrappers` API so we can add them exclusively here and remove them from all other places in the `CMakeLists.txt` file.

Reply:
Ok